src/ci/docker/x86_64-gnu-llvm-3.7/Dockerfile

```diff
@@ -26,5 +26,6 @@ ENTRYPOINT ["/usr/bin/dumb-init", "--"]
 
 ENV RUST_CONFIGURE_ARGS \
       --build=x86_64-unknown-linux-gnu \
-      --llvm-root=/usr/lib/llvm-3.7
+      --llvm-root=/usr/lib/llvm-3.7 \
+      --enable-test-miri
```

Comment:
Yes it will. On my machine it takes an additional 3 minutes (building + testing miri on 4 cores). It might take more due to the additional MIR that is generated, but last time I checked that it only caused more memory usage, not more (measurable) computation time.

I can probably reduce that time by making miri reuse rustc's compiletest.

Another 30s to 1 minute will be shaved off by the move of the miri base code to rustc (well, it will be inside rustc anyway, not actually reduced).